README.md

```diff
@@ -128,6 +130,9 @@ The `<branded-checkout>` element is where the branded checkout Angular app will
 - `on-order-completed` - an Angular expression that is executed when the order was submitted successfully - *Optional* -  provides 2 variables:
   - `$event.$window` - Provides access to the browser's global `window` object. This allows you to call a custom callback function like `onOrderCompleted` in the example.
   - `$event.purchase` - contains the order's details that are loaded for the thank you page
+- `radio-station-api-url` - Provides a URL path for fetching a list of radio stations in the donor's vicinity - *Optional*
```

Comment:
If we're adding a whitelist server-side, we will need to add instructions for those using these properties to contact us to add their API url to the whitelist.

Reply:
Should I mention a specific person/email to contact?

Reply:
Probably `help@cru.org` like the other place says.